cannon/testdata/example/mt-cond/main.go

```diff
@@ -0,0 +1,304 @@
+// Portions of this code are derived from code written by The Go Authors.
+// See original source: https://github.com/golang/go/blob/master/src/sync/cond_test.go
```

Comment:
Can you replace these with permalinks?:
```suggestion
// See original source: https://github.com/golang/go/blob/400433af3660905ecaceaf19ddad3e6c24b141df/src/sync/cond_test.go
```